fbgemm_gpu/codegen/training/forward/embedding_forward_split_kernel_v2_template.cu

```diff
@@ -738,6 +738,12 @@ __global__ void split_embedding_codegen_forward_{{ wdesc }}_v2_kernel(
     using emb_vec_t = vec4_type<emb_t>;
     using cache_vec_t = vec4_type<cache_t>;
     using output_vec_t = vec4_type<output_t>;
+#if defined(USE_ROCM)
+    using look_up_bits_t = uint64_t;
```

Comment:
why it needs to be uint64_t for rocm?

Reply:
This is done to account for ROCm 64 wave size (particularly, to get the right result from [bullot_sync](https://github.com/pytorch/FBGEMM/blob/89de1da1583303db0da9fbf7d9a3133ac7159ae1/fbgemm_gpu/include/fbgemm_gpu/utils/cuda_prelude.cuh#L111) function).